bm_scrapy/utils.py

```diff
@@ -6,3 +6,8 @@ def parse_time(date=None):
         date = datetime.now()
     parsed_time = date.strftime("%d/%m/%Y %H:%M:%S.%f")[:-3]
     return parsed_time
+
+
+def datetime_to_json(o):
+    if isinstance(o, datetime):
+        return o.__str__()
```

Comment:
Since we are using this as the default encoder, maybe we should raise a TypeError here in case it isn't a `datetime` object either?

Reply:
Added.